sdk/aws_orbit_sdk/controller.py

```diff
@@ -464,6 +464,29 @@ def list_storage_class():
     return res["items"]
 
 
+def get_nodegroups(cluster_name: str):
+    nodegroups: List[Dict[str, str]] = []
+    props = get_properties()
+    env_name = props["AWS_ORBIT_ENV"]
+    try:
+        cluster_name = f"orbit-{env_name}"
+        _logger.debug(f"Fetching cluster {cluster_name} nodegroups")
+        response: Dict[str, Any] = boto3.client("lambda").invoke(
+            FunctionName=f"orbit-{env_name}-eks-service-handler",
+            InvocationType="RequestResponse",
+            Payload=json.dumps({"cluster_name": cluster_name}).encode("utf-8"),
+        )
+        _logger.debug(response)
```

Comment:
remove these debug calls.. going to be hard to see anything if we have so much logging